doc/api/fs.md

```diff
@@ -1640,7 +1640,9 @@ Read data from the file specified by `fd`.
 `length` is an integer specifying the number of bytes to read.
 
 `position` is an integer specifying where to begin reading from in the file.
-If `position` is `null`, data will be read from the current file position.
+If `position` is `null`, data will be read from the current file position, 
+and the file position will be updated for subsequent reads.
```

Comment:
There is only one current file position, its used for read and for write, but this makes it sound like its used only for reads. I would just say the current file position is updated.